src/generateContent.js

```diff
@@ -4,17 +4,16 @@ import DEFAULT_OPTIONS from './defaultOptions';
  * @param {String} module
  * @param {String} locale
  * @param {Object} translations
- * @param {Boolean} mustRequireAngular
+ * @param {Boolean} requireAngular
  * @return {String}
  */
-export default (module = DEFAULT_OPTIONS.module, locale = DEFAULT_OPTIONS.defaultLocale, translations = {}, mustRequireAngular = DEFAULT_OPTIONS.requireAngular) => {
-    return `${mustRequireAngular ? 'var angular = require("angular");' : ''}
-var translations = ${JSON.stringify(translations, null, '\t')};
+export default (module = DEFAULT_OPTIONS.module, locale = DEFAULT_OPTIONS.defaultLocale, translations = {}, requireAngular = DEFAULT_OPTIONS.requireAngular) => {
+    return `var translations = ${JSON.stringify(translations, null, '\t')};
 var module;
 try {
-\tmodule = angular.module("${module}");
+\tmodule = ${requireAngular ? 'var angular = require("angular");' : 'window.'}angular.module("${module}");
```

Comment:
I was unsure whether I should have done it or not. I doubt that its not necessary. Do you think is it necessary?

Reply:
@Ardeshir81 I hope we should have it and without it we will break the code.

Reply:
@Ardeshir81 It is better to use:

```js
export default (module = DEFAULT_OPTIONS.module, locale = DEFAULT_OPTIONS.defaultLocale, translations = {}, requireAngular = DEFAULT_OPTIONS.requireAngular) => {
    return `var angular = ${requireAngular ? 'require("angular")' : 'window.angular'};
var translations = ${JSON.stringify(translations, null, '\t')};
var module;
try {
\tmodule = angular.module("${module}");
} catch(err) {
\tmodule = angular.module("${module}", ["pascalprecht.translate"]);
}
module.config(["$translateProvider", function($translateProvider) {
\t$translateProvider.translations("${locale}", translations);
}]);
module.exports = translations;`;
};
```

It will not break the test.

Reply:
yes! of course! its so much better. well done